akka-actor/src/main/scala/akka/pattern/RetrySupport.scala

```diff
@@ -30,17 +30,73 @@ trait RetrySupport {
    * private val sendReceiveRetry: HttpRequest => Future[HttpResponse] = (req: HttpRequest) => retry[HttpResponse](
    *   attempt = () => sendAndReceive(req),
    *   attempts = 10,
+   *   delay = 2 seconds)(
+   *    ec = context.system
+   *    scheduler = context.system.scheduler
+   * )
+   * }}}
+   */
+  def retry[T](attempt: () ⇒ Future[T], attempts: Int, delay: FiniteDuration)(implicit ec: ExecutionContext, scheduler: Scheduler): Future[T] =
```

Comment:
wouldn't `maxNrOfRetries` be a better name rather than `attempts`?

Reply:
Just trying to be consistent